config/jobs/cert-manager/cert-manager-presubmits.yaml

```
@@ -518,10 +517,11 @@ presubmits:
         - name: ndots
           value: "1"
 
+    # This is the default e2e test for all PRs.
   - name: pull-cert-manager-e2e-v1-23
```

```suggestion
  - name: pull-cert-manager-e2e-v1-23
    # This is the default e2e test for all PRs.
```

I'd argue that putting the comment _inside_ the yaml set more closely matches the code as of 1.22